modules/mobile/spec/spec_helper.rb

```diff
@@ -1,15 +0,0 @@
-# frozen_string_literal: true
-
-# Configure Rails Envinronment
-ENV['RAILS_ENV'] = 'test'
-# require File.expand_path('dummy/config/environment.rb', __dir__)
-
-require 'rspec/rails'
-
-ENGINE_RAILS_ROOT = File.join(File.dirname(__FILE__), '../')
-
-# Requires supporting ruby files with custom matchers and macros, etc,
-# in spec/support/ and its subdirectories.
-Dir[File.join(ENGINE_RAILS_ROOT, 'spec/support/**/*.rb')].sort.each { |f| require f }
-
-RSpec.configure { |config| config.use_transactional_fixtures = true }
```

Comment:
`spec_helper.rb` is not needed?

Reply:
Realized we want to use the core vets-api spec_helper so this one's not needed.